### PR TITLE
Do not mark values as secret that are from git-hosted sub config files

### PR DESCRIPTION
**Pull request by benbp**

### CODE REVIEW

File: eng/common/TestResources/SubConfig-Helpers.ps1 [hunk 1]
```diff
@@ -82,7 +118,8 @@ function ShouldMarkValueAsSecret([string]$serviceName, [string]$key, [string]$va
         "TenantId",
```
Comment (weshaggard) [inline eng/common/TestResources/SubConfig-Helpers.ps1:118]: Isn't the main values we want already in this list?

File: eng/common/TestResources/build-test-resource-config.yml [hunk 2]
```diff
@@ -61,24 +61,11 @@ steps:
         }
       }
 
+      Write-Host "Merging sub config from files"
```
Comment (weshaggard) [inline eng/common/TestResources/build-test-resource-config.yml:64]: Should we add other helper functions to clean-up this inline code?

**Review by weshaggard — APPROVED** [review]

File: eng/common/TestResources/build-test-resource-config.yml (hunk 2 above)
Comment (benbp) [reply eng/common/TestResources/build-test-resource-config.yml:64]: Yeah I was thinking about just eliminating the inline entirely. I'll go ahead and do that.

File: eng/common/TestResources/SubConfig-Helpers.ps1 (hunk 1 above)
Comment (benbp) [reply eng/common/TestResources/SubConfig-Helpers.ps1:118]: It's possible that in the storage case, the value is getting set from somewhere else to a config key that is not in this exclusion list. Was it the sub or the rg you were seeing starred out?

**Review by weshaggard — APPROVED** [review]

**Pull request merged**